src/memfs.ts

```diff
@@ -233,6 +232,11 @@ export class InMemoryFileSystem implements ts.ParseConfigHost, ts.ModuleResoluti
 	trace(message: string) {
 		this.logger.log(message);
 	}
+
+	// TODO: move this method to util and use through all the code
+	private path2uri(path: string): string {
```

Comment:
Here are few attempts to read files from memfs, see the mix of abs and relative paths:

```
? tsconfig.json
? C:/projects/javascript-typescript-langserver/node_modules/typescript/lib/lib.d
.ts
? d:/tmp/javascript-typescript-langserver/src/a.ts
```